cli/tests/test_completion.rs

```diff
@@ -242,3 +242,47 @@ fn test_remote_names() {
     );
     insta::assert_snapshot!(stdout, @r"origin");
 }
+
+#[test]
+fn test_aliases_are_completed() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["git", "init", "repo"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    // user config alias
+    test_env.add_config(r#"aliases.user-alias = ["bookmark"]"#);
+    // repo config alias
+    test_env.jj_cmd_ok(
+        &repo_path,
+        &[
+            "config",
+            "set",
+            "--repo",
+            "aliases.repo-alias",
+            "['bookmark']",
+        ],
+    );
```

Comment:
no action needed: I don't think we really need to test both user- and repo-level configs. The code doesn't seem to care about it, and it seems unlikely to regress.

Reply:
Makes sense, I did remove that test.